omnikinverter/tcp.py

```diff
@@ -29,18 +31,20 @@ class _AcOutput(BigEndianStructure):
         ("power", c_ushort),
     ]
 
-    def get_power(self) -> Optional[int]:
+    def get_power(self) -> int | None:
```

Comment:
Oh that's cool! (sorry for writing too much Rust :grimacing:)

Reply:
You can use this style for typing from python 3.10, but because of an import I added it also works with earlier versions.

Reply:
Ah so it's a new-ish feature entirely :)